docs/cookbook/README.md

```diff
@@ -8,4 +8,8 @@ Welcome to the cookbook. These code recipes show how to execute specific tasks,
 
 ## Hosting
```

Comment:
Based on the build, I believe this same info must also be added to the `.vuepress/sidebar.js` file in order to populate the page link in the documentation.